### DISCUSSION TITLE
Script per stoppare docker e riavviare supervisor

### DISCUSSION THREAD

**Discussion by scheccia1** [indomus/forum#3068] [HUB: Home Assistant]
Ciao,
mi chiedevo se è possibile fare eseguire uno script (collegandolo ad un pulsante) per stoppare il docker in modo da poter fare gli aggiornamenti... senza entrare via shell ogni volta.
Grazie

**Comment by snakuzzo** [comment]
Quello che chiedi è praticamente inutile.
Comunque...dai un'occhiata al live restore -> [QUI](https://docs.docker.com/config/containers/live-restore/)

**Reply by scheccia1** [reply]
perchè inutile? ogni volta che devo aggiornare home assistant devo collegarmi shell e stoppare il Portainer, riavviare il supervisor e poi posso aggiornare...
Forse mi sono spiegato male, il pulsante lo vorrei su home assistant in modo da lanciare i comandi senza entrare via shell al pc.

**Reply by snakuzzo** [reply]
Eh ? Scusa ma di cosa stai parlando ?
Tu nel titolo hai scritto "stoppare docker". Stoppare docker vuol dire fermare proprio il daemon di docker, non i container. Quindi forse hai sbagliato a scrivere.

Se intendevi l'aggiornamento di HA, se hai HA OS o HA Supervised, per aggiornare non devi fare niente. Devi semplicemente andare su aggiornamenti e farli cliccando...e non devi fermare niente.
